crates/wascc-provider/src/lib.rs

```diff
@@ -422,12 +432,73 @@ fn wascc_run(
     );
     capabilities.push(Capability {
         name: LOG_CAPABILITY,
+        binding: None,
         env: logenv,
     });
 
     let load = Actor::from_bytes(data).map_err(|e| anyhow::anyhow!("Error loading WASM: {}", e))?;
     let pk = load.public_key();
 
+    if load.capabilities().contains(&FS_CAPABILITY.to_owned()) {
```

Comment:
So what is happening here? Does the module have to specify that it needs an FS_CAPABILITY? Also, I think you can just do `FS_CAPABILITY` here instead of doing `to_owned` because it is already an `&str`

Reply:
Kubernetes attempts to inject a service account into each application runtime, so it is guranteed that each application will have at least one volume. For waSCC, if the application does not include a particular capability, then it will fail to run.

That's why we have to check if the actor has been signed with the ability to use the filesystem capability. If it has not signed off on the `wascc::blobstore` capability (like with the `greet`, `hello-world-assemblyscript` and `uppercase` demos), it will fail to load.